jenkins/catapult/baseImageDockerfileGenerator.py

```diff
@@ -129,9 +129,7 @@ def _zmq_descriptor(self):
 	def rocks(self):
 		descriptor = self.OptionsDescriptor()
 		descriptor.options += DEPENDENCY_FLAGS['facebook_rocksdb']
-
-		if 'undefined' in self.sanitizers:
-			descriptor.options += ['-DUSE_RTTI=1']
+		descriptor.options += ['-DUSE_RTTI=1']
```

Comment:
why is this unconditional now? not sure if we want it in release builds? 🤔

Reply:
@Jaguar0625 error message suggested, there's no way to compile _without_  it
see this https://jenkins.symboldev.com/job/Symbol/job/generated/job/symbol/job/catapult/job/fix%252Fupdate_dependencies_version/4/console

and I doubt we want two _different_ version of rocks...